Makefile

```diff
@@ -167,7 +167,11 @@ hooks: # Install Git hooks
 
 .PHONY: test
 test:
+ifndef RUN
```

Comment:
`RUN` is perhaps a bit too generic. Maybe `TEST_RUN` is a better name?

Reply:
Not quite sure we need this at all.

Reply:
Indeed this not a game changer but it helped to avoid running the whole test suite.

Reply:
It seems a good idea, but maybe for separate PR. :)